deployment/ansible/roles/cac-tripplanner.nginx/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+nginx_delete_default_site: False
```

Comment:
In addition to updating the nginx repo in the "Configure the Nginx PPA" task, I removed the line `nginx_version: "stable"` here in the copied over `defaults.yml` since it's no longer used by this role.